src/core/net/ip6_mpl.hpp

```diff
@@ -518,7 +518,7 @@ class Mpl
         kNumSeedEntries = OPENTHREAD_CONFIG_MPL_SEED_SET_ENTRIES,
         kSeedEntryLifetime = OPENTHREAD_CONFIG_MPL_SEED_SET_ENTRY_LIFETIME,
         kSeedEntryLifetimeDt = 1000,
-        kDataMessageInterval = 64
+        kDataMessageInterval = 200
```

Comment:
It violates Thread Specification. If with 64ms it works badly, maybe we should point it out in the Thread Group.